xCAT-test/autotest/testcase/infiniband/rhel-infiniband-diskfull-install.sh

```
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+########
+# Set all the variables below
+
+[ -n "$LINUX_DISTRO" ] &&
+LINUX_DISTRO="rhels7.4"
+[ -n "$LINUX_ARCH" ] &&
+LINUX_ARCH="ppc64le"
+
+[ -n "$COMPUTE_NODE" ] &&
+COMPUTE_NODE="c910f03c01p10"
```

It is not good idea to set a default value for ``COMPUTE_NODE``, it is dangerous. 
We will operate c910f03c01p10 accidentally.
``COMPUTE_NODE`` is necessary attribute, if it does not exist, script should return a error.